turborepo-tests/integration/tests/dry_json/monorepo.t

```diff
@@ -24,8 +24,8 @@ Setup
         ],
         "passThroughEnv": null
       },
-      "configured": [],
-      "inferred": [],
+      "configured": null,
+      "inferred": null,
```

Comment:
These look like unintentional changes due to Rust/Go differences in how empty/none gets serialized. Based on past experience it's worthwhile to try to avoid these slight changes in Rust migration.